.gitignore

```diff
@@ -25,3 +25,7 @@ dist/
 /node_modules
 .env
 /package-lock.json
+
+
+#database Migrations
+src/database/migrations
```

Comment:
You do not have to ignore migrations folder, this a crucial file while setting up a database